app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "forms" }) do |f| %>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
```

Comment:
- [ ] Great job working on setting up the devise but I think it would be great if you worked on the wireframe to match the given wireframe so kindly make the button width the same as the input width as shown in the given design 🔥💪🏻

- [OPTIONAL ] I highly recommend making the signup and login page the same airframe to improve from user experience (UX)
- Your design
![image](https://github.com/Lawmsangi/blog-app/assets/58553711/090e3e7d-a5af-4086-9083-b762d299cd68)

- Given
![image](https://github.com/Lawmsangi/blog-app/assets/58553711/cf5940e7-8ae2-4abb-a2f6-360339caa4a6)